react-components/src/components/Architecture/RevealButtons.tsx

```diff
@@ -4,24 +4,53 @@
 
 import { type ReactElement } from 'react';
 import { NavigationTool } from '../../architecture/base/concreteCommands/NavigationTool';
-import { CreateCommandButton } from './CommandButton';
+import { CommandButtonFromCommand } from './CommandButton';
 import { FitViewCommand } from '../../architecture/base/concreteCommands/FitViewCommand';
 import { FlexibleControlsType } from '@cognite/reveal';
 import { SetFlexibleControlsTypeCommand } from '../../architecture/base/concreteCommands/SetFlexibleControlsTypeCommand';
 import { SetAxisVisibleCommand } from '../../architecture/concrete/axis/SetAxisVisibleCommand';
 import { ClipTool } from '../../architecture/concrete/clipping/ClipTool';
 import { MeasurementTool } from '../../architecture/concrete/measurements/MeasurementTool';
+import { useFdmSdk } from '../RevealCanvas/SDKProvider';
+import { ObservationsTool } from '../../architecture/concrete/observationsDomainObject/ObservationsTool';
 
 export class RevealButtons {
-  static FitView = (): ReactElement => CreateCommandButton(new FitViewCommand());
-  static NavigationTool = (): ReactElement => CreateCommandButton(new NavigationTool());
-  static SetAxisVisible = (): ReactElement => CreateCommandButton(new SetAxisVisibleCommand());
-  static Measurement = (): ReactElement => CreateCommandButton(new MeasurementTool());
-  static Clip = (): ReactElement => CreateCommandButton(new ClipTool());
+  static FitView = (): ReactElement => (
```

Comment:
Please revert this. file. The commands doesn't need a common constructor argument. Beside this the coide complexity has increased a lot in this file.

Reply:
The constructor argument is necessary to avoid having the commands/tools reconstructed on every React re-render. But I can change back to using functions, but in that case we must be clear that they are not React-components, just regular functions that return React-components

Reply:
So I will change the casing to camel-case

Reply:
The result did not end up as clean as I had hoped, but I'm not sure how to do it better. If we can guarantee that we never want the tool that is attached to a button, to be reconstructed, we can make it a little simpler

Reply:
Went with that last idea. Ready for re-review